packages/block-library/src/image/style.scss

```diff
@@ -195,14 +195,28 @@
 	}
 
 	.wp-block-image {
-		display: flex;
-		justify-content: center;
-		align-items: center;
 		width: 100%;
 		height: 100%;
-		z-index: 3000000;
 		position: absolute;
+		z-index: 3000000;
+		box-sizing: border-box;
+		display: flex;
+		justify-content: center;
+		align-items: center;
 		flex-direction: column;
+		padding: 20px;
+
+		figcaption {
+			margin: 10px 0 0 0;
+		}
+
+		img {
+			max-width: 100%;
+			max-height: 100%;
+			width: auto;
+			min-height: 0;
+			min-width: 0;
```

Comment:
Do we need this? Is not 0 the default value?

Reply:
Oh, I read [here](https://www.w3schools.com/cssref/pr_dim_min-height.php#:~:text=Show%20demo%20%E2%9D%AF-,Default%20value%3A,-0) that the default value is true, and removing those lines didn't seem to affect. That's why I thought they weren't necessary.